### DISCUSSION TITLE
What features would you like to see in this site?

### DISCUSSION THREAD

**Discussion by Accio1** [theforumhelpers/theforumhelpers.github.io#37] [Q&A]
This discussion is set up as a Q&A, so feel free to propose feature ideas below. Then, other people can vote for the suggestion that they think would be the most beneficial. This will help determine the priority in which something should be worked on, as well as allowing discussion for things that people would like to be added, as well as the best way to implement them.

**Comment by leahcimto** [comment]


**Reply by Accio1** [reply]
Can you provide examples/mockups of what you mean?

**Comment by leahcimto** [comment]
and a applications open/closed indicator

**Reply by Accio1** [reply]
This is listed as a to do in the site overhaul discussion here: https://github.com/theforumhelpers/theforumhelpers.github.io/discussions/35

**Comment by leahcimto** [comment]
one more- a page with news and updates

**Reply by Accio1** [reply]
Since there isn't much news/updates aside from changes to the application process, this will be low priority in the site overhaul.

**Comment by leahcimto** [comment]
I also think it would be cool if we could display post count with the list somehow (like with ScratchDB.)

**Reply by Accio1** [reply]
I agree. Incorporating this with shefwrld or ScratchDB is a good idea in order to give some statistics about each person.

**Comment by qbjl** [comment]
host this at a website like https://theforumhelpers.ga or something

**Reply by Accio1** [reply]
This site is not developed by, or intended for people in Gabon, so I don't see why a .ga domain would be appropriate.

**Reply by Steve0Greatness** [reply]
I think that was one of the domains they found on google domains.

I just found out that theforumhelpers.com isn't taken, that means that they just chose 2 random letters.
